### ISSUE TITLE
POST requests fail for local invocations

### ISSUE THREAD

**Issue by mdklatt** [mdklatt/pytest-cgi#2]
The ``LocalClient.post()`` method fails with:
```
TypeError: expected str, bytes or os.PathLike object, not int
```

The problem is the ``CONTENT_LENGTH`` environment variable, which is defined as an ``int``:

[cgi.py](https://github.com/mdklatt/pytest-cgi/blob/master/src/pytest_cgi/cgi.py#L93)
``` python
       env = {
            "REQUEST_METHOD": "POST",
            "CONTENT_LENGTH": len(data),
            "CONTENT_TYPE": mime,
        }
        self._call(env, data)
```

An ``env`` dict passed to ``subprocess.run()`` must only contain string values. This does not fail during unit testing because ``subprocess.run()`` is mocked.

**Comment by mdklatt** [comment]
Commit 8f054ec0af196c7613e29accba8b3ce82d465e5e should fix this, need to test with a real-world project to verify it.

**Comment by mdklatt** [comment]
Fix works with a real-world project, need a better mock for `subprocess.run()` in testing.

**Comment by mdklatt** [comment]
Issue resolved by commit 82bcc80f63a867215f4adadd66a82d0d3ae33b45.
